composer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "moxie-leean/wp-elements",
```

Comment:
We should update package as well @zzuga  example: 

https://packagist.org/packages/moxie-lean/wp-endpoints-collection